utils/test/TestKickStart.bat

```diff
@@ -0,0 +1,4 @@
+
+@ECHO ON
+python TestRunner.py
```

Comment:
... or just add it as a comment and we can figure it out later. Rather have you concentrate on tying it all together.